Cargo.toml

```diff
@@ -11,6 +11,7 @@ cli = ["comfy-table", "clap"]
 enable-serde = ["serde", "serde_json"]
 protocol-perf-testing = ["no-prss"]
 no-prss = []
+disable-metrics = []
```

Comment:
I wanted to profile with and without metrics. Metrics takes 15% of the peak heap allocation and is the most significant one in IPA. If it will be enabled in helpers, we may need to consider dropping some metrics to reduce its memory consumption.

100 rows, t-gmax
with metrics: 113,474,349 bytes
w/o metrics: 97,055,576 bytes

Reply:
I would prefer to keep them - metrics are essential part of infrastructure and the visibility into their overhead is crucial to improve. Totally fine if there is a switch, but I think we should keep the metrics on by default. 

I suspect they suffer from the same step issue and should not have big overhead on top of that. If that's the case, then we have a problem...

Reply:
OK, removed `disable-metrics` from `dhat-heap` feature.